src/CSAccounting.sol

```diff
@@ -57,6 +59,7 @@ contract CSAccounting is
     constructor(
         address lidoLocator,
         address module,
+        address _feeDistributor,
```

Comment:
it's ok to have `feeDistributor` here

Reply:
Nope, cause we have a view function with the same name